plugins/catalog/src/components/UnregisterEntityDialog/UnregisterEntityDialog.tsx

```diff
@@ -82,23 +98,36 @@ export const UnregisterEntityDialog = ({
       <DialogTitle id="responsive-dialog-title">
         Are you sure you want to unregister this entity?
       </DialogTitle>
+
       <DialogContent>
         {loading ? <Progress /> : null}
+
         {error ? (
           <Alert severity="error" style={{ wordBreak: 'break-word' }}>
-            {error.toString()}
+            {error instanceof DeniedLocationException ? (
+              <>
+                You cannot unregister this entity, since it originates from a
+                protected Backstage configuration (location
+                {`"${error.locationName}"`}). If you believe this is in error,
+                please contact your Backstage operator.
```

Comment:
haha i just realized ... there's a config key to get the app name isn't there :) that we are supposed to use

Reply:
We could also introduce a config key for some kind of operator contact 🤔

Reply:
It's `app.title`. Could then be `If you believe this is in error, please contact the __Backstage Example App__ operators.`.